scripts/run_celery.sh

```diff
@@ -2,8 +2,29 @@
 
```

Comment:
currently nothing is using this script, we split into `run_celery_no_sms_sending.sh` and `run_celery_send_sms.sh` basically.

Doesn't hurt to leave this in here, but we should add this code to those other two scripts as well.

Reply:
Whoops. Thanks, I added it to the other script.